corehq/apps/geospatial/static/geospatial/js/geospatial_map.js

```diff
@@ -35,21 +39,50 @@ hqDefine("geospatial/js/geospatial_map", [
                     trash: true
                 },
             });
-            map.addControl(draw);
 
-            map.on('draw.create', updateArea);
-            map.on('draw.delete', updateArea);
-            map.on('draw.update', updateArea);
+            map.addControl(draw);
 
             function getCoordinates(event) {
                 return event.lngLat;
             };
 
-            function updateArea(e) {
-                const data = draw.getAll();
-                const area = turf.area(data);
-                // Restrict the area to 2 decimal points.
-                const rounded_area = Math.round(area * 100) / 100;
+            map.on("draw.update", function(e) {
+                var selectedFeature = e.features[0];
```

Comment:
Can we assume that there will always be a first element? If not, it might be better to rewrite this to something like:
```
if (!e.features.length) {
    return;
}
var selectedFeature = e.features[0];
...
```

Reply:
I think we can, but good to have the check. I'll add this. This event might fire in other cases, but if the polygon is updated by being moved, then the polygon feature is the only one existing here, which is what we assume.